style.css

```diff
@@ -551,3 +556,375 @@ a.btn-see-resume:active {
 .no-padding {
   padding: 1px;
 }
+
+@media only screen and (min-width: 992px) {
+  .desk-menu {
+    display: flex;
+    padding-left: 10px;
+  }
+
+  .head-wrapper {
+    height: 50px;
+    margin-left: 155px;
+    margin-right: 155px;
+  }
+
+  .nav-bar {
+    padding-top: 25px;
+  }
+
+  .desktop-menu {
+    width: 200px;
+    margin: 0 25px 0 25px;
+    position: static;
+    align-items: center;
+    text-align: center;
+    display: flex;
+    padding-left: 0;
+  }
+
+  .top-menu {
+    font-weight: 500;
+    font-size: 15px;
+    font-style: normal;
+    font-family: 'Poppins', sans-serif;
+    line-height: 20px;
+    padding-left: 12px;
+    padding-right: 10px;
+    margin: 0 10px;
+  }
+
+  .top-menu a {
+    color: #344563;
+    text-decoration: none;
+  }
+
+  .first-menu {
+    font-weight: 500;
+    font-size: 15px;
+    font-style: normal;
+    font-family: 'Poppins', sans-serif;
+    line-height: 20px;
+    padding-right: 10px;
+    padding-left: 12px;
+    margin: 0 10px;
+  }
+
+  .first-menu a {
+    color: #344563;
+    text-decoration: none;
+  }
+
+  .toogle-btn {
+    display: none;
+  }
+
+  .headline-wrapper {
+    border-bottom-left-radius: 110px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-image: url("assets/Header_desktop.png");
+    width: 1440px;
```

Comment:
- [ ] [OPTIONAL] You have done an excellent job addressing the previous reviewer suggestions :muscle: but there is still an issue with the background size because it's not covering the entire width. I think the issue is because you are applying a fixed width for this section. I will mark this as [OPTIONAL] because I think it won't take you much to fix this but I strongly suggest addressing this issue for a better user experience :wink:

  ![image](https://user-images.githubusercontent.com/81522435/144439020-8dfec05e-e401-4491-b760-1587ab7734f3.png)